CHANGELOG.md

```diff
@@ -1,217 +1,241 @@
+: provides demo backends and backend validation utils for your project
 
-Instead of competing with the full page cart feature, it only contains a subset of actions, such as removing an item and checking out.
-For additional modifications to cart products, it links to the cart page.
+`venia-sample-language-packs`
+: provides demo translations
 
-Developers can still use the old `MiniCart` component in their projects, but it has been renamed to `LegacyMiniCart`.
+[`packages/extensions`]: <https://github.com/magento/pwa-studio/tree/release/9.0/packages/extensions>
 
-### Branding updates
+### Internationalization and localization
 
-The UX team continues in their research to improve the look and feel of the Venia brand.
-This release includes many style updates to give the Venia storefront a more modern and accessible experience.
+This releases introduces the Internationalization(I18n) feature in PWA Studio.
+This lets developers localize their storefront content according to different regions and languages.
 
-Developers get all these improvements right away when they start their projects using this version of Venia as the base storefront or by upgrading their dependencies.
+This release refactors Venia UI components to use this feature and adds components that support multi-language storefronts.
 
-### My Account code preview
+This release also gives developers the ability to develop and install PWA Studio language packages as NPM dependencies.
+An example of a language pack extension can be found in the [`packages/extensions`][] directory in the PWA Studio repository.
 
-This releases includes a sneak peak at features associated with My Account, such as Wishlist, Order History, and Address Book.
-Even though the Communications Page is the only navigable page, curious developers can peek at the 8.0.0 release codebase to see the initial code for these features.
+For more information about this feature, read the new [Localization topic][] on the doc site.
 
-### Targets reference documentation
+[localization topic]: <https://pwastudio.io/technologies/basic-concepts/internationalization/>
 
-During 8.0.0 development, the PWA Studio doc site has published reference documentation for extensibility targets in the different packages.
-This documentation contains API descriptions and sample code to help developers discover the different PWA Studio extension points.
+### My Account
+
+This release adds components that support _My Account_ features for customers that create an account with a store.
+
+My Account features included in this release:
+
+- Wishlist
+- Saved Payments
+- Address Book
+- Order History
+
+Work on the following My Account features are in progress and currently unavailable:
+
+- Store Credit and Gift Cards
+- Favorites Lists
+
+### Increased test coverage
+
+As part of our initiative to increase stability and confidence in the PWA Studio project, this release adds new unit tests to existing code to improve code coverage.
```

Comment:
This could be better worded I believe

> Our continued commitment to stability and quality has seen an increase in overall unit test code coverage....